tests/math/test-isna.py

```diff
@@ -74,8 +74,10 @@ def test_isna_joined():
     dt.internal.frame_integrity_check(RES)
     assert RES.to_list() == [[True, True, False, True, False]] * 4
 
-
+@pytest.mark.xfail(reason="scalar NA check not supported")
```

Comment:
Hm, was it ever supported? If yes, we probably need to restore it. I didn't notice we had this test.

Reply:
yeah, I cant see the python fallback for this though

Reply:
Then we need to restore this functionality, who knows may be it is used by some projects.

Reply:
So it seems all the unary functions can take a python scalar as the argument: https://github.com/h2oai/datatable/blob/main/src/core/expr/funary/pyfn.cc#L66-L93

We need to see how to implement that with the new `FExpr`s.